plugins/reach/lib/model/data/VendorVideoAnalysisCatalogItem.php

```diff
@@ -20,4 +23,33 @@ public function isEntryTypeSupported($type, $mediaType = null): bool
 		$supportedMediaTypes = [entry::ENTRY_MEDIA_TYPE_VIDEO];
 		return $type === entryType::MEDIA_CLIP && in_array($mediaType, $supportedMediaTypes);
 	}
+
+	public function setVideoAnalysisType($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_VIDEO_ANALYSIS_TYPE, $v);
+	}
+
+	public function getVideoAnalysisType()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_VIDEO_ANALYSIS_TYPE, null, true);
+	}
+
+	public function setMaxVideoDuration($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_MAX_VIDEO_DURATION, $v);
+	}
+
+	public function getMaxVideoDuration()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_MAX_VIDEO_DURATION, null, true);
```

Comment:
the default values doesnt makes sense to me here as well

Reply:
fixed